exampleCourse/questions/demo/addHTMLtable/server.py

```diff
@@ -10,7 +10,9 @@ def generate(data):
     mytable = (
         '<table style="width:30%"><tr><th> Name </th> <th> Age </th> <th> Id </th></tr>'
     )
-    for name, age, name_greek in zip(name_list, age_list, name_greek_list):
+    for name, age, name_greek in zip(
+        name_list, age_list, name_greek_list, strict=False
```

Comment:
Do we actually want `strict=False` here? AFAICT we do want these to always be the same lengths. Ditto for some other lines in this PR.